package.json

```diff
@@ -45,6 +45,6 @@
     "build": "babel src --out-dir .",
     "check": "node_modules/.bin/eslint --ext \".js\" --ignore-pattern node_modules --quiet .",
     "test": "NODE_ENV=test mocha --opts test/mocha.opts",
-    "postinstall": "npm run build"
+    "prepublishOnly": "npm run build"
```

Comment:
I believe removing the postinstall script will cause issues when using the library as a non-npm module, at least for me in the mobile apps